deploy/010_serviceaccount-rolebinding.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sre-ebs-iops-reporter
+  namespace: openshift-monitoring
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sre-ebs-iops-reporter
+  namespace: openshift-monitoring
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: edit
+subjects:
+- kind: ServiceAccount
+  name: sre-ebs-iops-reporter
+  namespace: openshift-monitoring
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: sre-allow-read-machine-info
```

Comment:
Inconsistent name, suggest all objects have `ebs-iops-reporter` in it, in this case it would be `sre-ebs-iops-reporter-read-machine-info` maybe.

Note prefix of `sre-` is not consistent for all objects.  Decide what to do there per https://github.com/openshift/managed-prometheus-exporter-dns/pull/1#pullrequestreview-214019177 then changes need to apply here.

Reply:
This is addressed in 5320231.